toolchains/sysimage/BUILD.bazel

```diff
@@ -15,14 +15,22 @@ exports_files([
     "verity_sign.py",
 ])
 
+py_library(
+    name = "container_utils",
+    srcs = ["container_utils.py"],
+    deps = [
+        requirement("invoke"),
+    ],
+)
+
 # Creates a tar file as output
 py_binary(
     name = "build_container_base_image",
     srcs = [
         "build_container_base_image.py",
-        "container_utils.py",
     ],
     deps = [
+        ":container_utils",
         requirement("invoke"),
```

Comment:
this is a transitive dep right? can we remove it from here? same in the other target